desc/objectives/objective_funs.py

```diff
@@ -130,10 +158,44 @@ def build(self, use_jit=None, verbose=1):
             self._scalar = False
 
         self._set_derivatives()
+        sub_obj_jac_chunk_sizes = [obj._jac_chunk_size for obj in self.objectives]
+        warnif(
+            np.any(sub_obj_jac_chunk_sizes) and self._deriv_mode != "blocked",
+            UserWarning,
+            "'jac_chunk_size' was passed into one or more sub-objectives, but the"
+            "ObjectiveFunction is  using 'batched' deriv_mode, so sub-objective "
+            "'jac_chunk_size' will be ignored in favor of the ObjectiveFunction's "
+            f"'jac_chunk_size' of {self._jac_chunk_size}."
+            " Specify 'blocked' deriv_mode if each sub-objective is desired to have a "
+            "different 'jac_chunk_size' for its Jacobian computation.",
+        )
+        warnif(
+            self._jac_chunk_size is not None and self._deriv_mode == "blocked",
+            UserWarning,
+            "'jac_chunk_size' was passed into ObjectiveFunction, but the"
+            "ObjectiveFunction is using 'blocked' deriv_mode, so sub-objective "
+            "'jac_chunk_size' are used to compute each sub-objective's Jacobian, "
+            "`ignoring the ObjectiveFunction's 'jac_chunk_size'.",
+        )
```

Comment:
I have a question about this. If say the user wants to use `blocked` in a multiobjective optimization with one of the `jac_chunk_size` specified and the rest of the `jac_chunk_size` different than `dim_x/4`, wouldn't it be easier to set the `jac_chunk_size` of sub-objectives to the `jac_chunk_size` of the ObjectiveFunction?
So, like
```python
obj = ObjectiveFunction(
    [
    Objective_1(eq, jac_chunk_size=30),
    Objective_2(eq),
    Objective_3(eq),
    ],
    deriv_mode="blocked",
    jac_chunk_size=10,
)
```
The natural way I understand this, the user wants specifically `jac_chunk-size=30` but 10 for the rest, not the default value.

Reply:
IF blocked is used, I want the default to be whatever is put for the sub objective. It is a choice, i'd rather be verbose than start having some not entirely clear logic for what sizes are used where